decidim/decidim
# Fix errors when building documentation

#### :tophat: What? Why?

These last few weeks I've been updating [Antora](https://docs.antora.org/antora/latest/), the static generator that we use for our documentation site (docs.decidim.org).

In the process we have a new feature for free: when building the site we see the errors and warnings, such as broken links, images, etc.  

This PR fixes some of these, and also introduces a couple of changes that I've seen while I was at it. Most notably:

*  [Add SMS service page](https://github.com/decidim/decidim/commit/4664febfc06d3ded23bc4d34e2f7fe9858f433ad)
*  [Rename 'How to fix metrics' to 'Troubleshooting metrics' and link from the 'Metrics' page](https://github.com/decidim/decidim/commit/800e878d85b993e5b1ee4d37f3e3718c40aecec7)
*  [Add 'Customize' index page and move generator instructions to this page ](https://github.com/decidim/decidim/commit/342f18b720cecc328de481f0fb14b383f48c73fc)

#### :pushpin: Related Issues
 
- Related to  https://github.com/decidim/documentation/runs/6572272105?check_suite_focus=true
- As we're adding a new page, after merging this we need to merge https://github.com/decidim/documentation/pull/101

#### Testing

1. Clone https://github.com/decidim/documentation/ 
2. Follow the README to install it
3. Build to see the output (`npm run build`)
4. Clone this branch locally
5. Change the `antora-playbook.yml` file to match your local environment with branches, something like:
```diff
diff --git a/antora-playbook.yml b/antora-playbook.yml
index a4728c0..0e5f088 100644
--- a/antora-playbook.yml
+++ b/antora-playbook.yml
@@ -8,9 +8,9 @@ content:
       start_path: /en
       branches: master
       edit_url: "https://github.com/decidim/documentation/edit/{refname}/{path}"
-    - url: https://github.com/decidim/decidim
+    - url: /home/apereira/Work/decidim/decidim
       start_path: docs
-      branches: develop
+      branches: docs/fix-warnings-errors
     - url: https://github.com/decidim/decidim-bulletin-board
       start_path: docs
       branches: develop
```
6. Build to see the output (`npm run build`)

:hearts: Thank you!
